pkg/controller/node/node_controller.go

```diff
@@ -963,6 +970,10 @@ func (ctrl *Controller) updateCandidateMachines(pool *mcfgv1.MachineConfigPool,
 		if err := ctrl.setDesiredMachineConfigAnnotation(node.Name, targetConfig); err != nil {
 			return goerrs.Wrapf(err, "setting desired config for node %s", node.Name)
 		}
+		ctrl.logPool(pool, "Applying %s taint to %s", constants.NodeUpdateInProgressTaint.Key, node.Name)
+		if err := ctrl.setUpdateInProgressTaint(node.Name); err != nil {
+			return goerrs.Wrapf(err, "failed applying %s taint for node %s", constants.NodeUpdateInProgressTaint.Key, node.Name)
```

Comment:
nit: the wrapper and log are both making some assumptions about what `setUpdateInProgressTaint` is up to.  Probably pretty safe assumptions, so feel free to ignore, but it would be slightly more robust to push those down into `setUpdateInProgressTaint` itself.

Reply:
I thought, we usually error or log but not both in fn or method. IIUC, you want log to be present in `setUpdateInProgressTaint` method?

Reply:
what ravi did here seems consistent with the func call above

Reply:
tho @wking is there a specific change to the error message above you think would be better?